### PR TITLE
Added public and private uploading

### PR DESCRIPTION
**Pull request by chxcolate**
Added public and private uploading through two javascript files:

./build/private.js
./build/public.js

They can be run using:

npm run private
npm run public

### CODE REVIEW

File: build/public.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        modules[name] = content;
+      } else {
+        modules[name] = base64_encode(filename);
+      }
+
+      next();
+    }
+  },
+  function () {
+    /* We now have our modules object, now we just need
```
Comment (WilWa) [inline build/public.js:43]: Should we remove this comment?

**Review by WilWa — CHANGES_REQUESTED** [review]
See comments.

**Comment by chxcolate** [comment]
Sure

**Review by WilWa — APPROVED** [review]
Awesome!

**Pull request merged**